server/lib/live-blogs/transform-live-blogs.js

```diff
@@ -0,0 +1,12 @@
+'use strict';
+
+const cheerioTransform = require('../cheeerio-transform');
```

Comment:
This isn't quite right ;)  The requires on lines 6 and 7 now also need updating - suspect live blogs need testing against Alphaville come Monday